src/main/java/wellnus/reflection/GetCommand.java

```diff
@@ -131,7 +132,7 @@ public void validateCommand(HashMap<String, String> commandMap) throws BadComman
             throw new BadCommandException(INVALID_COMMAND_MSG);
         } else if (!commandMap.containsKey(COMMAND_KEYWORD)) {
             throw new BadCommandException(INVALID_COMMAND_MSG);
-        } else if (!commandMap.get(COMMAND_KEYWORD).equals(PAYLOAD)) {
+        } else if (!(commandMap.get(COMMAND_KEYWORD).equals(PAYLOAD))) {
```

Comment:
Is there really a need to wrap this in ()? For sake of standardization, this means ALL `!condition` should be rewritten as `!(condition)`